documentation/book/proc-creating-new-image-from-base.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// assembly-using-kafka-connect-with-plugins.adoc
+
+include::common/attributes.adoc[]
+
+[id='creating-new-image-from-base-{context}']
+= Create a new image based on our base image
+
+{ProductName} provides its own Docker image for running Kafka Connect, which can be found on {DockerRepository} as
+`{DockerKafkaConnect}`.
+This image can be used as a base image for building a new custom image with additional plugins.
+
+The following procedure describes the process for creating such a custom image.
+
+.Procedure
+
+. Create a new `Dockerfile` using `{DockerKafkaConnect}` as the base image:
++
+[source,subs="+quotes,attributes"]
+----
+FROM {DockerKafkaConnect}
+USER root:root
+COPY ./_<my-plugins>_/ /opt/kafka/plugins/
+USER {DockerImageUser}
+----
+
+. Build the container image and upload it to the appropriate container image repository.
+
+. Use the new container image in the Kafka Connect deployment:
+.. On {OpenShiftName}, the `KafkaConnect.spec.image` property of the KafkaConnect custom resource and the `STRIMZI_DEFAULT_KAFKA_CONNECT_IMAGE` variable can be changes to point to the new image once the Kafka Connect cluster is deployed.
```

Comment:
@scholzj can you confirm that I did this right? I put the links in the additional resources section below.